src/pymedphys/cli/main.py

```diff
@@ -68,6 +70,15 @@ def dicom_cli(subparsers):
     dicom_adjust_rel_elec_density_parser.set_defaults(
         func=adjust_rel_elec_density_cli)
 
+    dicom_structure_name_RED_adjust = dicom_subparsers.add_parser(
+        'structure-name-red-adjust')
```

Comment:
Please change "red" to "RED".

Reply:
Done, see https://github.com/pymedphys/pymedphys/pull/93/files#diff-e72b2d3c62c8b4b520c73962fc042c4eR74